src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -158,6 +159,16 @@ const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange,
                 tooltip: 'Browse bucket files'
               }, [icon('folder-open', { size: 20 })])
             ]),
+            !readOnly && h(Link, {
+              style: { marginLeft: '0.25rem' },
+              onClick: () => {
+                const json = Utils.maybeParseJSON(value)
+                if (json !== undefined) {
+                  onChange(name, JSON.stringify(json, null, 2))
+                }
+              },
+              tooltip: 'Format'
```

Comment:
Since this is only for JSON formatting I wonder if a more specific tool tip would make sense? Something like 'format JSON' ?

Reply:
For the wording, I wrestled with that a bit. While it is technically JSON, I don't know if we call that out specifically, and a non-programmer might be confused by it. I also considered 'Format struct', and while that would certainly make sense to a WDL author, I'm not sure if that would be clear to someone just running it.

Regarding disabling/feedback, it's a good point. I had originally skipped that due to worries about performance (since we'd need to pre-parse all the inputs to generate a good disabled flag), but now that we've tightened things up a bit, maybe we can revisit that.

Reply:
Oh okay that's a good point about the wording, leaving it to just "format" sounds like a safe approach. 

Looking again I think what made the functionality confusing for me was that even when I clicked the format button the tool tip stayed visible and there is no visual/color change to the button so I couldn't be sure if I was really clicked or just hovering. 

No 'on click' visual change appears to be the norm across Terra but usually a click opens/closes something so there is some sort of visual clue something happened. Disabling does seem like it could cause a performance hit (especially when there are like 60 inputs) I wonder if maybe we could add a visual cue just to the on-click? (example: https://material.io/components/buttons/#)

Reply:
In some places, specifically with copy to clipboard buttons, we briefly replace the button with a checkmark, maybe something like that?